locales/it/era.yml

```diff
@@ -1,38 +1,37 @@
 ---
 it:
-  language: language
-  interviewers: interview team
-  cameramen: camera team
-  date_of_interview: date of interview
-  alias_name: alias name
+  language: lingua
+  interviewers: intervistatori
+  cameramen: riprese
+  date_of_interview: data dell'intervista
+  alias_name: pseudonimo
   begin_of_engagement:
   end_of_engagement:
-  resistance_groups: resistance groups
-  kind_of_resistance: kind of resistance
-  firstname: firstname
-  lastname: lastname
-  gender: gender
-  born_on: birthdate
-  died_on: died on
+  resistance_groups: gruppi di resistenza
+  kind_of_resistance: tipo di resistenza
+  firstname: nome
+  lastname: cognome
+  gender: genere
+  born_on: data di nascita
+  died_on: data di morte
   born_in: born in %{date}
```

Comment:
since we use it this way we can do `GIACOMO NOTARI (CLASSE 1927)` similar to `annata`